vite.config.ts

```
@@ -1,7 +1,8 @@
 import { VitePWA } from 'vite-plugin-pwa';
-import { defineConfig } from 'vite'
+import { defineConfig, PluginOption } from 'vite'
```

ESLint

[error] 'PluginOption' is defined but never used.

(@typescript-eslint/no-unused-vars)